.github/workflows/build.yml

```diff
@@ -35,6 +35,8 @@ jobs:
 
     - name: Build python-build (without isolation)
       run: python -m build -x -n
+      env:
+        PYTHONPATH: src
```

Comment:
This defeats half the purpose of using a `src/` environment. We should only be testing _installed_ versions of the library, since that's what end users will be using.

If we want to test the ability to bootstrap `build` from the local directory, we should do that separately.

Reply:
I definitely want to test running from a local directory, it will be required for bootstrapping after `setup.py install` gets removed. The question is which behavior we should keep in this PR. Both behaviors will be tested in the integration tests, so I don't have a strong opinion here, though I am inclined to just set `PYTHONPATH`.

Reply:
I think the situations in which we support running from a local directory should be *extremely* limited, and we should have custom, explicit tests for them if we test them at all.

You should never be installing `Solaar` or `python-dateutil` using `build` installed in a local directory, so we definitely shouldn't be testing it.

We should drop *all* path manipulation in all of our tests right now, and if we want to introduce a separate "can you bootstrap `build`" (or some other legitimate use for running it from a local directory), we should do that in a different PR.